go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.6.0
+	github.com/btcsuite/btcd v0.21.0-beta // indirect
```

Comment:
It is a bit strange that the `go.mod` file changed but not the `go.sum`. Could you run `go mod tidy` on your branch and push again?